spec/factories/measure_factory.rb

```diff
@@ -16,7 +16,7 @@
     end
 
     geographical_area do
-      attributes_for(:geographical_area, id: geographical_area_id)
+      attributes_for(:geographical_area, id: geographical_area_id, geographical_area_id:)
```

Comment:
The api responding with 3 geo area ids is a bit mad but clearly historical

Reply:
Yeh, just figured I'd fix up the factory - confused me initially